supernovacontroller/sequential/gpio.py

```diff
@@ -0,0 +1,212 @@
+from transfer_controller import TransferController
+from BinhoSupernova.Supernova import Supernova
+from BinhoSupernova.commands.definitions import (
+    GpioPinNumber, GpioLogicLevel, GpioFunctionality, GpioTriggerType,
+)
+from supernovacontroller.errors import BackendError
+from sequential import SupernovaDevice
```

Comment:
This import is not being used

Reply:
Furthermore, this breaks the example for me, I get
```
ModuleNotFoundError: No module named 'sequential'
```
If I delete this line, then it works as expected

Reply:
Good catch!! It was added accidentally in [this commit](https://github.com/binhollc/SupernovaController/pull/44/commits/f17659815ff35507db40c07b2b9068aa7c1b204e).